source/whats-new.txt

```diff
@@ -64,46 +64,50 @@ To learn more, see the `v4.12.0 Release Highlights <https://github.com/mongodb/n
 What's New in 4.11
 ------------------
 
-The 4.11 {+driver-short+} release includes **recursive schema support**,
-meaning that you can use recursive types as the schema in the
-``Collection`` class. The driver also provides type safety for dot
-notation queries up to a depth of nine in this release. Typescript compiles
-your code beyond a depth of nine, but types fall back to ``any`` at
-this level. The recursive type depth limit is a current limitation in
-TypeScript.
+The 4.11 {+driver-short+} release includes added support for **mutually 
+recursive** collection schema types. The driver also provides type safety for 
+dot notation queries up to a depth of eight in this release. Typescript compiles 
```

Comment:
v4.11 release notes say nine, but tests of original PR say [otherwise](https://github.com/mongodb/node-mongodb-native/pull/3433/files#:~:text=%7D)%3B-,//%20Depth%20of%208%20does%20**not**%20have%20type%20checking,-expectAssignable%3CFilter)